### ISSUE TITLE
Not clear GridLayout issue

### ISSUE THREAD

**Issue by shabib3** [NativeScript/NativeScript#5310]
Hi all :)

I need some UI help here :slightly_smiling_face:
I have a not clear GridLayout issue
first column element doesn't fill al the space but the column is set with * param.

here is a playground sample that will tell the issue:
[Playground project](https://play.nativescript.org/?template=play-ng&id=E1hE6B)

in the HTML I wrote exactly what I am looking for.

Thank you very much!

**Comment by NickIliev** [comment]
@shabib3 because the top parent GridLayout (the one after `*ngFor` ) has a **rows** set to **auto** and will span to take only the space needed not the space available.

Here is [a working demonstration ](https://play.nativescript.org/?template=play-ng&id=E1hE6B&v=5)where your third stack will be placed in the bottom of  the cell

**Comment by shabib3** [comment]
@NickIliev 
So first of all I implemented the ListView element instead of heavy view with `*ngFor` for another issue I have and I got to say thank you very much it is awesome! the performance just crazy right now! :)

About this issue...the solution you gave me doesn't work on my side...And I think the moment we will get the GridLayout in the Playground sample not stretch all over the page and fill only `auto` it will happen on my side too.
